pymc3/distributions/timeseries.py

```diff
@@ -309,8 +311,14 @@ def _random(self, sigma, mu, size, sample_shape):
         else:
             axis = 0
         rv = stats.norm(mu, sigma)
-        data = rv.rvs(size).cumsum(axis=axis)
-        data = data - data[0]  # TODO: this should be a draw from `init`, if available
+        if len(size) == 1:
+            data = rv.rvs(size).cumsum(axis=axis)
+            data = data - data[0]  # TODO: this should be a draw from `init`, if available
+        else:
+            data = np.empty(size)
```

Comment:
May be safer to do `np.empty(size, dtype=theano.config.floatX)`

Reply:
Thank you for pointing this out